README.md

```diff
@@ -197,6 +198,17 @@ jwt.verify(token, cert, { algorithms: ['RS256'] }, function (err, payload) {
   // if token alg != RS256,  err == invalid signature
 });
 
+// Verify using getKey callback
+function getKey(header, callback) {
```

Comment:
In order to use this function for JWK + openid cases we would need to add the issuer as well.

The options are:
* We add it here like `getKey(header, issuer, callback)`, and think if we want to check the issuer, if passed in options, before even calling the `getSecret` function.
* Leave it as is in this PR. Give support for that in a different one by checking the length of `getKey` function (to see if the caller expects only header and callback or header, issuer and callback)

What do you think?

Reply:
For the sake of progress, i would like to keep it as is. Let's make an separate issue for it and i will pick it up after this one ;-)